browser-test/src/support/applicant_questions.ts

```diff
@@ -157,12 +157,44 @@ export class ApplicantQuestions {
     index = 0,
   ) {
     await this.page.fill(`.cf-date-year input >> nth=${index}`, year)
+
+    // Empty string means "delete this answer". The dropdown default is "Select"
+    if (month == '') {
+      month = 'Select'
+    }
+
     await this.page.selectOption(`.cf-date-month select >> nth=${index}`, {
       label: month,
     })
     await this.page.fill(`.cf-date-day input >> nth=${index}`, day)
   }
 
+  async checkMemorableDateQuestionValue(
+    year: string,
+    month: string,
+    day: string,
+    index = 0,
+  ) {
+    const yearValue = await this.page
```

Comment:
**suggestion (non-blocking):** You could consider extracting this into a helper function, since its pretty similar for year / month / day values.

Reply:
I think the code would be simpler to understand like this, even though it's more verbose. Is that OK?

Reply:
Sounds good!